### PR TITLE
Add clearDetectedCache flag to StartDetectionThread

### PR DESCRIPTION
**Pull request by braineo**

### CODE REVIEW

File: mujinvisioncontrollerclient/visioncontrollerclient.py [hunk 1]
```diff
@@ -230,6 +231,8 @@ def StartDetectionThread(self, vminitparams, regionname=None, cameranames=None,
             command['destregionname'] = destregionname
         if ignoreBinpickingStateForFirstDetection is not None:
             command['ignoreBinpickingStateForFirstDetection'] = bool(ignoreBinpickingStateForFirstDetection)
+        if clearDetectedCache is not None:
```
Comment (woodychow) [inline mujinvisioncontrollerclient/visioncontrollerclient.py:234]: I am against this kind of None check. This argument should not be allowed to be None.

Comment (rdiankov) [reply mujinvisioncontrollerclient/visioncontrollerclient.py:234]: None means you let the defaults decide, i think it is ok....
it is better than having `null` getting through to JSON

**Comment by rdiankov** [comment]
thanks~

File: mujinvisioncontrollerclient/visioncontrollerclient.py (hunk 1 above)
Comment (woodychow) [reply mujinvisioncontrollerclient/visioncontrollerclient.py:234]: It's an optional argument. You can just ignore it...

**Pull request merged**